workalendar/tests/test_europe.py

```diff
@@ -98,6 +99,28 @@ def test_year_2016(self):
         self.assertIn(date(2016, 12, 26), holidays)  # St. Stephen's Day Prvi d
 
 
+class Cyprus(GenericCalendarTest):
+    cal_class = Cyprus
+
+    def test_year_2017(self):
+        holidays = self.cal.holidays_set(2017)
+        self.assertIn(date(2017, 1, 1), holidays)    # New Year's Day
```

Comment:
It's not blocking IMHO. When I can get some time, I'll have to change every test to check against a set(), that'll make sure there are no extra or missing days